pkgs/development/compilers/go/1.10.nix

```diff
@@ -179,7 +179,7 @@ stdenv.mkDerivation rec {
     homepage = http://golang.org/;
     description = "The Go Programming language";
     license = licenses.bsd3;
-    maintainers = with maintainers; [ cstrahan orivej wkennington ];
+    maintainers = with maintainers; [ cstrahan orivej velovix mic92 ];
```

Comment:
@velovix are you ok with that?

Reply:
Sure, that's fine

Reply:
Do we know that wkennington isn't interested in being a maintainer anymore?

Reply:
@velovix he has its fork of nixpkgs, he is working: https://github.com/wkennington?tab=overview&from=2017-12-01&to=2017-12-31